src/app/store/step/step.store.ts

```diff
@@ -192,4 +192,10 @@ export class StepStore {
   getSteps() {
     return this.steps;
   }
+
+  // Check if we need a custom form handling which stores the parsed
+  // properties in customProperties
+  isCustomStep(step: Step) : boolean {
+    return step.stepKind === BASIC_FILTER;
```

Comment:
The data mapper should fall into this category too really.  If we set an empty `configuredProperties` on the step object in that case it shouldn't be the end of the world.  In which case we can reduce the code a bit more in the step-configure controller, WDYT?